### PR TITLE
replaymod replay crash fix 1.17+ 

### PR DESCRIPTION
**Pull request by wagyourtail**
use world not worldData to prevent npe when in replaymod replay

oopsie while merging up

### CODE REVIEW

File: src/main/java/baritone/utils/BlockStateInterface.java [hunk 1]
```diff
@@ -97,9 +97,9 @@ public BlockState get0(BlockPos pos) {
     }
 
     public BlockState get0(int x, int y, int z) { // Mickey resigned
-        y -= worldData.dimension.minY();
+        y -= world.dimensionType().minY();
```
Comment (wagyourtail) [inline src/main/java/baritone/utils/BlockStateInterface.java:100]: does JIT optimize these function calls away, or should I move this to a `private final int`

dimensionType just returns a final field
minY is also just returning a final field

Comment (Martine1231) [reply src/main/java/baritone/utils/BlockStateInterface.java:100]: can you explain how you fixed this in low IQ speech. I appreciate it

Comment (wagyourtail) [reply src/main/java/baritone/utils/BlockStateInterface.java:100]: world -> worldData as world can sometimes be null

Comment (Martine1231) [reply src/main/java/baritone/utils/BlockStateInterface.java:100]: Capture
checking through but I cant find a file for worldData

Comment (wagyourtail) [reply src/main/java/baritone/utils/BlockStateInterface.java:100]: :rock: :brain:

**Comment by EvilEpicPanda** [comment]
I'm confused. Issues get closed citing this issue. Is there a fix for this or are we waiting for it to be fixed?

**Comment by wagyourtail** [comment]
well, has there been a release of baritone for (insert version you still experience this on) since feb 8 2022 when this was fixed

**Comment by EvilEpicPanda** [comment]
> well, has there been a release of baritone for (insert version you still experience this on) since feb 8 2022 when this was fixed

Yes. I was using 1.9.4, latest release.

**Comment by SunfxDEV** [comment]
Im also using 1.9.4 with a modpack and im getting the same errors, any fix?

**Pull request merged**